docs/prod-host-setup.md

```diff
@@ -46,6 +46,20 @@ for consuming and storing this data safely. We suggest using any upper-bounded
 forms of storage, such as fixed-size or ring buffers, programs like `journald`
 or `logrotate`, or redirecting to a named pipe.
 
+### Logging and performance
+
+There is extra latency introduced in Firecracker functionality as the process
+generates and logs various error or warning messages. Some host configurations
+can have an effect on the performance of the process as Firecracker will
+generate host kernel logs during normal operations. The most recent example of
+this was the addition of `console=ttyAMA0` kernel command line argument on one
```

Comment:
I would specify clearly that we're talking about the host kernel.

Reply:
Done.